<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerEditor</class>
 <widget class="QDialog" name="LayerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit layer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblDelay">
       <property name="text">
        <string>Delay</string>
       </property>
       <property name="buddy">
        <cstring>spinDelay</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="sliderDelay">
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinDelay">
       <property name="maximum">
        <number>4096</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblUnk1">
       <property name="text">
        <string>Unknown 1</string>
       </property>
       <property name="buddy">
        <cstring>spinUnk1</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinUnk1">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblUnk1Hex">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>0x0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblBendLow">
       <property name="text">
        <string>Bend Range [-]</string>
       </property>
       <property name="buddy">
        <cstring>spinBendLow</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderBendLow">
       <property name="maximum">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBendLow">
       <property name="maximum">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblBendHigh">
       <property name="text">
        <string>Bend Range [+]</string>
       </property>
       <property name="buddy">
        <cstring>spinBendHigh</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="sliderBendHigh">
       <property name="maximum">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinBendHigh">
       <property name="maximum">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblUnk2">
       <property name="text">
        <string>Unknown 2</string>
       </property>
       <property name="buddy">
        <cstring>spinUnk2</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinUnk2">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="lblUnk2Hex">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>0x0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LayerEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LayerEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderDelay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinDelay</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinDelay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderDelay</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBendHigh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBendHigh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBendHigh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderBendHigh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBendLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBendLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBendLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderBendLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
